lib/menu_loader.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './dish'
+class MenuLoader
+  def initialize
+    @objects = []
+  end
+  def load(file)
+    $/="\n\n"
+    File.open(file, "r").each do |object|
+      @objects << YAML.load(object)
+    end
+    objects
+  end
+  def objects
+    @objects.dup
+  end
+end
```

Comment:
I don;t see the need for a menu loader - this is again a symptom of going beyond the brief - there was no mention of a persistent data model i.e. yaml file.